logstash-core/spec/logstash/json_spec.rb

```diff
@@ -56,6 +56,20 @@
         expect(LogStash::Json.dump(array)).to eql(json_array)
       end
 
+      context "pretty print" do
+
+        let(:hash) { { "foo" => "bar", :zoo => 2 } }
+
+        it "should serialize with pretty print" do
+          pprint_json = LogStash::Json.dump(hash, :pretty => true)
+          expect(pprint_json).to include("\n")
```

Comment:
I'm not sure if doing it strictly could actually leak to some specific implementation details, should all pretty print be the same? or just include CR? I mean for example in the indentation. For now I think I will leave it like this and we can always improve it later on.